docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md

```diff
@@ -159,14 +159,88 @@ Here are some examples:
  
- For CRDs installed in system world, by default it's only accessible for tenant users. But if it's marked with "multi-tenancy.k8s.io/published" annotation, it will be accessible to all tenants.
+ 1. **isolation**. Each regular tenant can independently create/delete/update their CRDs and custom resources based on the CRDs. There will be NO CRD G/V/K collision between two regular tenants. Actually, the CRDs and custom resources of one regular tenant are invisible to another regular tenant.  
+
+ 2. **Managebility**. A cluster operator (he should be user under the system tenant) can introduce a new CRD, where he can choose to:
+  * make it usable only to the system tenant.
+  * or, make it usable only to a spefic set of to regular tenants and the system tenants, 
+  * or, usable to all the regular tenants. 
+  * Additionally, the applicable range of a CRD can be changed by the cluster operator at any time in a simple and quick manner. That will be useful to CRD test and rolling update.  
  
- If there is a collision between CRDs in system world and tenant world, the latter one takes precedence if it's accessed by a tenant.
+ Note: it does not contradict the principle of isolation. The isolation principle is about two regular tenants, while this principle is about the system tenant and regular tenants. 
+
+ 3. **Autonomy**. The tenant admin has the final say on what CRD will be used in his tenant space when there is an overlap between the tenant's CRD and that of the system tenant. The tenant can choose MultiTenancyCRD policy to be :
+  * NeverUseSystemCRD
+  * SystemCRDFirst
+  * LocalCRDFirst
+
+  Besides, the tenant user can change the MultiTenancyCRD policy anytime. 
+
+  The default policy will be LocalCRDFirst. Under the default policy, if a tenant has deployed CRD objects and CRD operators based on its own CRD, the introduction of an overlapping system CRD will not cause break.
+
+ 4. **Backward compatibility**. The legacy CRD definitions and CRD operators continue to work in Arktos. More specifically, it means:
+    * An existing defintion of CRD (usually in a .yaml file) can be applied without any change.
+    * An existing CRD operator image can work in Arktos without re-compiling or image rebuilding.
+    * An existing CRD operator source code build and work in Arktos. So developers can create new operators or revise existing operators in Arktos without learning new APIs.
+
+However, we need to point out that, Arktos does not allow a regular tenant to access cluster-scope resources, such as nodes and daemonsets. So a CRD definition or a CRD operator that needs access to the cluster-scope resources will no longer be supported in a regular tenant's space. Surely, they can continue to work in the system tenant's space. For the same reason, a regular tenant is not allowed to create a cluster-scope CRD. If regular tenant specifies its CRD to be cluster-scoped, the CRD will be changed to tenant-scoped. 
```

Comment:
Suggest rewording to something like "certain resources are not accessible from regular tenants, like node objects".   "Arktos does not allow a regular tenant to access cluster-scope resources..." is too strong since there are many cluster-scoped resources.

Reply:
updated as suggested.